app/scripts/directives/overview/serviceInstanceRow.js

```diff
@@ -39,10 +39,30 @@
       return _.get(row, ['state','serviceClasses', serviceClassName, 'description']);
     };
 
+    var updateProvisionStatus = function() {
+      var conditions = _.get(row.apiObject, 'status.conditions');
+      var readyCondition = _.find(conditions, {type: "Ready"});
+
+      row.provisionError = _.find(conditions, {type: "Failed"});
+
+      if (_.get(row.apiObject, 'metadata.deletionTimestamp')) {
+        row.provisionStatus = 'deleted';
+      } else if (row.provisionError) {
+        row.provisionStatus = 'error';
+      } else if (readyCondition && readyCondition.status === 'True') {
+        row.provisionStatus = 'ok';
+      } else {
+        row.provisionStatus = 'inProgress';
+      }
+    };
+
     row.$doCheck = function() {
+      updateProvisionStatus();
+
       row.notifications = ListRowUtils.getNotifications(row.apiObject, row.state);
       row.displayName = serviceInstanceDisplayName(row.apiObject, row.state.serviceClasses);
-      row.isBindable = BindingService.isServiceBindable(row.apiObject, row.state.serviceClasses);
+      row.isBindable = !row.provisionError &&
+                       BindingService.isServiceBindable(row.apiObject, row.state.serviceClasses);
```

Comment:
I'd rather check the `Failed` condition in `BindingService.isServiceBindable`, but I know that is a common change. OK as a follow on.